examples/metrics/src/main.rs

```diff
@@ -89,12 +91,12 @@ fn main() -> Result<(), Box<dyn Error>> {
 
 /// Our network behaviour.
 ///
-/// For illustrative purposes, this includes the [`keep_alive::Behaviour`]) behaviour so the ping actually happen
+/// For illustrative purposes, this includes the [`dummy::Behaviour`]) behaviour so the ping actually happen
 /// and can be observed via the metrics.
```

Comment:
That isn't quite true :)

The `dummy::Behaviour` does nothing so you can remove it from this struct. This comment should move to the `idle_connection_timeout()` function call and we should set it to a much higher value, like 1 minute or something. See, what `keep_alive::Behaviour` used to do here is keep all connections alive indefinitely. That would allow `ping::Behaviour` to keep pinging and this example would showcase the metrics collected as a result. Now we are replacing `keep_alive::Behaviour` with a different mechanism :)

Let me know if this makes sense!